homework_19/conftest.py

```diff
@@ -0,0 +1,27 @@
+import pytest
+from selenium.webdriver import Chrome
+
+from homework_19.dashboard2 import Dashboard2
+from homework_19.dashboard1 import Dashboard1
+
+
+@pytest.fixture(scope="session")
+def driver() -> Chrome:
+    # PATH should be yours for driver
+    driver = Chrome("C:/Users/Admin/PycharmProjects/Voitiuk-AQA-Selenium-Python/drivers/chromedriver.exe")
```

Comment:
Will work only on windows. Only your windows) so you should add path to PATH variable previously. Take a look on os.setenv function.